server/chat-commands/info.ts

```diff
@@ -2413,52 +2416,126 @@ export const commands: ChatCommands = {
 	],
 
 	showimage(target, room, user) {
-		if (!target) return this.parse('/help showimage');
-		if (!this.can('declare', null, room)) return false;
-		if (this.room.isPersonal && !this.user.can('announce')) {
-			return this.errorReply(`Images are not allowed in personal rooms.`);
-		}
-
-		const targets = target.split(',');
+		return this.errorReply(`/showimage has been deprecated - use /show instead.`);
+	},
 
-		if (targets.length !== 1 && targets.length !== 3) {
-			return this.parse('/help showimage');
+	requestapproval(target, room, user) {
```

Comment:
I would call this `requestshow`, and I would default the approvals system to off.